### ISSUE TITLE
Expand CI for Mobile

### ISSUE THREAD

**Issue by ravinderjangra** [maidsafe/safe_app_csharp#33]
CI Platform: Azure DevOps (formally VSTS)

Why these changes are required?
- Currently we are running only .Net Core tests on CI (AppVeyor). To make sure the bindings are working fine on Android and iOS, these tests should also run on CI.
- This improvement will help us to detect any issue which may occur while updating tests/bindings/libs/APIs.

**Use Cake script to achieve following goals :**
- Build and run desktop(.Net Core) tests project
- Build and run Android test project on ARM emulator API-25
- Build and run iOS test project using simulator
- Upload test results for all platforms on DevOps 
- Fail build in case of test failure.

**Comment by S-Coyle** [comment]
This project board seems to be for documentation only (according to description) - should this card be elsewhere @ravinderjangra ?

**Comment by ravinderjangra** [comment]
yes @S-Coyle this board is for documentation mainly but I planned to complete this CI task before this release so I put this task under same board,
